### ISSUE TITLE
[Testnet Demo] Faucet transaction never confirms and link broken

### ISSUE THREAD

**Issue by markmhendrickson** [leather-io/extension#929]
I'm stuck at the current state waiting for Stacks tokens perpetually when testing https://github.com/blockstack/ux/pull/761:

![image](https://user-images.githubusercontent.com/28991/105712335-ea9e5c00-5f19-11eb-9be0-5b7e78a59b94.png)

The "View transaction in Explorer" link goes [here](https://testnet-explorer.blockstack.org/txid/0x3a8d1000c046998ed1bc362fc8c7149a2d6d7e57c2353bf2eb0498f839f4c268) but immediately re-routes to the Explorer homepage. So it appears the transaction was never generated successfully?

We should handle error handling here in addition to assuring that transactions are generated. I've noticed the app gets stuck in this state frequently, so perhaps we should remove it as an upfront blocker in general and instead handle "lack of STX" UX upon initiate of transaction signing later in the UX, since we need it there any case for the average app.

**Comment by timstackblock** [comment]
This is still not working

<img width="1434" alt="Screen Shot 2021-02-01 at 12 06 58 PM" src="https://user-images.githubusercontent.com/49165468/106492566-11aedd80-6486-11eb-8ddb-1ce711768ee7.png">


<img width="1436" alt="Screen Shot 2021-02-01 at 12 05 44 PM" src="https://user-images.githubusercontent.com/49165468/106492393-ddd3b800-6485-11eb-9626-7eb7c852c00e.png">

**Comment by hstove** [comment]
Hey @timstackblock - what you're reporting is a separate issue. However, the bug you're seeing is because 761 is no longer the 'official' release PR. Please use https://github.com/blockstack/ux/pull/936 . This is my fault for not communicating.

**Comment by andresgalante** [comment]
@timstackblock would you mind confirming that it's fixed and close the issue, please?

**Comment by timstackblock** [comment]
Please see the screen shot below, @andresgalante this cannot be tested and is blocked by the bug above. No matter what version of the test app you use the canary build of the extension returns the error above when using the testnet link https://stacks-node-api.testnet.stacks.co. 

Is there another URL for the testnet we can point to that will not give us this error.

<img width="1439" alt="Screen Shot 2021-02-02 at 11 50 41 AM" src="https://user-images.githubusercontent.com/49165468/106633587-f6f36c00-654c-11eb-9da4-966300ab0d43.png">

**Comment by andresgalante** [comment]
@hstove should this be fixed before the audit?

**Comment by hstove** [comment]
Since there are a few things going on:

- You can still sign in to the app. You'll need to switch networks (in the top right menu) to mainnet. Then you can sign in (I just tested that). The default network is mainnet, too.
- You are blocked from transaction signing because the testnet is down, so you can't QA the actual bug here. Tomorrow LMK when you want to test this, and I'll give you a node URL.

As a sanity check, I've just tested this using the Canary release + demo app, and was able to do STX transfers successfully.

**Comment by markmhendrickson** [comment]
I just tried this but testnet appears still down so I'm getting this error when changing my network to testnet:

![image](https://user-images.githubusercontent.com/28991/106742524-6df83580-661d-11eb-93f2-8820501cec3c.png)

I do think we need to resolve this one way or the other before the audit, so I'm going to assign it to that milestone. @andresgalante could you keep your eye on this as we approach next Wednesday and work with devops as needed to best ensure availability then? 

- The #stacks-2-testnet channel in Discord is where chatter about uptime/downtime takes place, and where we can submit requests to the team when it goes down
- http://status.test-blockstack.com/ is a quick reference site for seeing the status of the testnet
- Loading the Stacks Blockchain API instance run by PBC, and as configured against testnet, here should also not throw a 502 when it's back up https://stacks-node-api.testnet.stacks.co
